openreview/tools.py

```diff
@@ -152,18 +155,37 @@ def get_profiles(client, ids_or_emails, with_publications=False):
 
     profiles = []
     profile_by_email = {}
+    profiles_as_dict = {}
 
     batch_size = 100
     for i in range(0, len(ids), batch_size):
         batch_ids = ids[i:i+batch_size]
         batch_profiles = client.search_profiles(ids=batch_ids)
         profiles.extend(batch_profiles)
 
+    if as_dict:
+        profiles_by_name = {}
+        for profile in profiles:
+            for name in profile.content.get("names", []):
+                profiles_by_name[name.get("username")] = profile
```

Comment:
name.get("username") can return None here, is it ok to have a None key?

Reply:
Yeah, it won't cause any issues.